analyses/interaction-plots/scripts/03-plot_interactions.R

```diff
@@ -203,34 +206,60 @@ if (is.na(opts$disease_table)) {
-
-# Previously calculated top 10
-# display_diseases <- disease_df %>%
-#   dplyr::select(disease, mutant_samples) %>%
-#   dplyr::arrange(desc(mutant_samples)) %>%
-#   dplyr::select(disease) %>%
-#   unique() %>%
-#   head(10) %>% # top 10 diseases with highest mutated samples
-#   dplyr::pull(disease)
-
-# Now a manual version
-display_diseases <- c(
-  "Diffuse midline glioma",
-  "Low-grade glioma astrocytoma",
-  "Craniopharyngioma",
-  "High-grade glioma astrocytoma",
-  "Ganglioglioma",
-  "Medulloblastoma",
-  "Meningioma",
-  "Ependymoma",
-  "Schwannoma",
-  "Dysembryoplastic neuroepithelial tumor"
-)
-
-disease_df <- disease_df %>%
-  # remove disease == NA, these are samples where
-  # harmonized_diagnosis is Benign tumor, Dysplasia/Gliosis
+  dplyr::mutate(gene = factor(gene, levels = genes)) %>%
+# add plot header rows
+  add_row(gene = "TP53", mutant_samples = 0.00001, disease = "High-grade gliomas") %>%
+  add_row(gene = "TP53", mutant_samples = 0.00001, disease = "Low-grade gliomas")
+
+# What are the top 10 mutated cancer display groups?
+ display_diseases <- disease_df %>%
+   # remove other from top 10 possibilities
+   dplyr::filter(disease != "Other") %>%
+   dplyr::select(disease, mutant_samples) %>%
+   dplyr::arrange(desc(mutant_samples)) %>%
+   dplyr::select(disease) %>%
+   unique() %>%
+   head(10) %>% # top 10 diseases with highest mutated samples
+   dplyr::pull(disease)
+ 
+ # Print
+ print(display_diseases)
+ 
+ # We want to set the order to have "other" HGG or LGG come last within the groups
+ 
+ display_diseases <- c("High-grade gliomas",
+                       "Diffuse midline glioma",
+                       "Other high-grade gliomas",
+                       "Low-grade gliomas",
+                       "Pilocytic astrocytoma",
+                       "Ganglioglioma",
+                       "Pleomorphic xanthoastrocytoma",
+                       "Other low-grade gliomas",
+                       "Medulloblastoma",
+                       "Atypical Teratoid Rhabdoid Tumor",
+                       "Other embryonal tumors",
+                       "Ependymoma",
+                       "Craniopharyngioma",
+                       "Meningioma")
+ 
+ display_disease_lab <- c(expression(bold("High-grade gliomas")),
+                       "Diffuse midline glioma",
+                       "Other high-grade gliomas",
+                       expression(bold("Low-grade gliomas")),
+                       "Pilocytic astrocytoma",
+                       "Ganglioglioma",
+                       "Pleomorphic xanthoastrocytoma",
+                       "Other low-grade gliomas",
+                       "Medulloblastoma",
+                       "Atypical Teratoid Rhabdoid Tumor",
+                       "Other embryonal tumors",
+                       "Ependymoma",
+                       "Craniopharyngioma",
+                       "Meningioma", 
+                       "Other")
```

Comment:
Adding in Embryonal comment, and using `atop` to add blank lines between groupings
```suggestion
                       expression(atop(" ", bold("Low-grade gliomas"))),
                       "Pilocytic astrocytoma",
                       "Ganglioglioma",
                       "Pleomorphic xanthoastrocytoma",
                       "Other low-grade gliomas",
                       expression(atop(" ", bold("Embryonal tumors"))),
                       "Medulloblastoma",
                       "Atypical Teratoid Rhabdoid Tumor",
                       "Other embryonal tumors",
                       expression(atop(" ", "Ependymoma")),
                       "Craniopharyngioma",
                       "Meningioma", 
                       "Other")
```